integration_tests/suite/test_google_contacts.py

```diff
@@ -160,17 +183,25 @@ def test_list(self, google_api):
                 items=contains_inanyorder(
                     has_entries(
                         name='Mario Bros',
+                        given_name='Mario',
+                        family_name='Bros',
                         emails=contains_inanyorder('mario@bros.example.com'),
+                        email_labels=contains_inanyorder('other'),
                         numbers=contains_inanyorder('+15555551111', '+15555551234'),
                         numbers_by_label=has_entries(
                             home='+15555551111', mobile='+15555551234'
                         ),
+                        organizations=contains_inanyorder('MarioLand'),
+                        organization_titles=contains_inanyorder('Artist'),
+                        addresses=contains_inanyorder('Main Land', 'Alternative Land'),
+                        address_labels=contains_inanyorder('home', 'Second address'),
```

Comment:
After IRL discussion, labels should be merged with objects inside a list of dictionnary, not a list of string